plugins/coverbrowser.koplugin/main.lua

```diff
@@ -132,14 +132,14 @@ function CoverBrowser:addToMainMenu(menu_items)
             separator = true,
         },
         {
-            text = _("Shorten home directory to ~"),
+            text = _("Shorten home directory to Home"),
```

Comment:
> Since the file browser can be considered "Home"

_Can be_, but not _must be_ :) So don't _must_ me!
That name can be tweaked, as the setting is just the value to be shown.
I don't mind replacing the default with `Home`, but you should allow me to use `~/`. (OK, one could manually hack it into its settings.reader.lua - and if one has already checked that box, it will stay with `~` - until one play again with that checkbox.)
So, this should be some InputText (so no checkbox? or a checkbox for callback - and a hold_callback to set the name to use)?
Also, `text = _("Shorten home directory"),` , or include the setting value.

And "Home" should be translatable.
And I dunno. It looks nice in english with Home (`Home/Wikipedia/Science`) - may be less with some french translation (`Accueil/Wikipedia/Science`), and probably bothering in languages with long words (`MyLilleHomeInThePrairie/Wikipedia/Science`).

Reply:
This change I'm impartial with, I like Home personally but it would be nice to allow the user to set the text, e.g. ~ or Home, I will have to do some more digging to figure out how to do that as I am still learning how all the code for KOReader is structured, haven't worked with LUA before either